src/features/document/use_cases/remove_by_path_use_case.py

```diff
@@ -1,14 +1,12 @@
-from typing import Optional
-
 from authentication.models import User
+from common.utils.string_helpers import split_dmss_ref
 from services.document_service import DocumentService
 from storage.internal.data_source_repository import get_data_source
 
 
-def remove_by_path_use_case(
-    user: User, directory: str, data_source_id: Optional[str] = None, repository_provider=get_data_source
-):
+def remove_by_path_use_case(user: User, absolute_path: str, repository_provider=get_data_source):
```

Comment:
Is there a difference between `id_reference`and `absolute_path`? If not, can use id_reference here for consistency.
(For example in the remove_use_case.py, the function takes in `id_reference` and it looks like that is the same.)